### DISCUSSION TITLE
How do I use the FM Radio to play sounds.

### DISCUSSION THREAD

**Discussion by Millymilo00** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1605] [Q&A] [answered]
Found the "FM Radio" somewhere, and I quickly found out online that if you put in a specific string of numbers it'll play a song. In that same page I that said that, it also said something about playing your own noteblock sounds with numbers. I've tried messing around with it myself but I can't seem to figure it out whatsoever.

Can anyone help me figure this out?

**Comment by OsakaLinuxScraper** [comment]
The ID Should be 2012-08-06 
It plays "Song of storms - Legend of Zelda"

**Reply by Millymilo00** [reply]
Yes I'm aware, I vaguely mentioned that in my original post, I probably should've been more clear.
But anyway what I was really curious about, is how I can play my own noteblock sounds or just anything else in general using the FM Radio.

**Reply by OsakaLinuxScraper** [reply, marked as answer]
We dont know really yet.

I could go into the FM Radio Source Code(i have a local copy of the source code which auto-updates every release), and look.
And.. all it does at the moment is.. Well. Play one Singular Sound.

Sorry for the late reply, i check github only monthly.

**Reply by Millymilo00** [reply]
Ah thats a shame, I guess I've been misled... Oh well, would be really cool if we could play our own stuff on the FM radio.

**Reply by asdwastakenig** [reply]
New Acc.. I have already made a fork of the mod, still working on it. maybe ill add it but id annoy HBM until you get banned or he does it. (prob wont.)
